### PR TITLE
feat (Python binding): Add output builders

### LINKED ISSUE #903: Output builder for python bindings

### PR DESCRIPTION
**Pull request by bingyanglin**
# Description of change

Add output builder for ease of generating OutputDto in Python

## Links to any relevant issues

Fixes #903 #889 

## Type of change

Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Local testing.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/python/native/iota_client/client.py [hunk 1]
```diff
@@ -19,6 +19,74 @@ def __init__(self, client_config=None):
     def get_handle(self):
         return self.handle
 
+    @staticmethod
+    def create_treasury_output(amount):
```
Comment (Thoralf-M) [inline bindings/python/native/iota_client/client.py]: That's not needed, since a user can't create it in the network

Comment (bingyanglin) [reply bindings/python/native/iota_client/client.py]: Yes so I marked them as static methods. It's a bit redundant. I will remove them in the python codes.

Comment (bingyanglin) [reply bindings/python/native/iota_client/client.py]: Removed at [0988abc](https://github.com/iotaledger/iota.rs/pull/943/commits/0988abc98ecae2304a8c54c1655915fba895433f)

File: bindings/python/native/src/lib.rs [hunk 2]
```diff
@@ -48,10 +55,346 @@ pub fn send_message(handle: &ClientMessageHandler, message_type: String) -> Resu
     Ok(serde_json::to_string(&response)?)
 }
 
+#[pyfunction]
+#[allow(clippy::too_many_arguments)]
+/// Create OutputDto::Treasury JSON string
+pub fn create_treasury_output(amount: u64) -> Result<String> {
```
Comment (Thoralf-M) [inline bindings/python/native/src/lib.rs]: Here it's also useless

Comment (bingyanglin) [reply bindings/python/native/src/lib.rs]: To make it consistent so I added it :)

Comment (bingyanglin) [reply bindings/python/native/src/lib.rs]: I removed it at [31ee5c1](https://github.com/iotaledger/iota.rs/pull/943/commits/31ee5c11fd25b70e0bf75f94634592f68da67ace)

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**